frontend/components/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "vue-strap",
+  "version": "2.14.1",
+  "description": "Bootstrap components built with Vue.js",
+  "keywords": [
+    "bootstrap",
+    "vue-bootstrap",
+    "vue-component",
+    "vue",
+    "markbind"
+  ],
+  "homepage": "https://markbind.org",
+  "license": "MIT",
+  "files": [
+    "LICENSE",
+    "README.md",
+    "CHANGELOG.md",
+    "src",
+    "dist"
+  ],
+  "main": "dist/vue-strap.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MarkBind/markbind.git"
+  },
+  "scripts": {
+    "build": "webpack --progress --hide-modules --config  webpack.build.js",
+    "test": "jest",
+    "updatesnapshot": "jest --updateSnapshot",
+    "builddev": "webpack --progress --hide-modules --config  webpack.dev.js",
```

Comment:
Alphabetical order?

Reply:
resolved. perhaps we could add https://www.npmjs.com/package/sort-package-json as a dev dependency at another time 😅